components/editor/prosemirror-plugins.md

```diff
@@ -0,0 +1,134 @@
+---
+title: ProseMirror Plugins
+page_title:  ProseMirror Plugins
+description: Explore how to use the ProseMirror Plugins in the Editor for Blazor uses.
+slug: editor-prosemirror-plugins
+tags: telerik,blazor,editor,prosemirror,plugins
+published: True
+position: 130
+---
+
+# ProseMirror Plugins
+
+The Telerik UI for Balzor Editor component is based on the [ProseMirror library](https://prosemirror.net/). ProseMirror provides a set of tools and concepts for building rich text editors, using user interface inspired by what-you-see-is-what-you-get.
```

Comment:
```suggestion
The Telerik UI for Blazor Editor component is based on the [ProseMirror library](https://prosemirror.net/). ProseMirror provides a set of tools and concepts for building rich text editors, using user interface inspired by what-you-see-is-what-you-get.
```

Reply:
"user interface inspired by what-you-see-is-what-you-get." does not sound very English to me, so I would remove it.

Note that we may use "rich text editor" for [another component](https://feedback.telerik.com/blazor/1435752-rich-text-editor).